lib/events.js

```diff
@@ -171,6 +172,15 @@ exports.editEvent = async (req, res) => {
             core.fetchBody(body, req.headers['x-auth-token'])));
     }
 
+    // TODO fix this so the status get changed upon saving
```

Comment:
@serge1peshcoff do you have any idea how to do this properly? I would like to not copy the setApprovalStatus function in here

Reply:
what do you want to do here?

Reply:
I want to change the approval status to first/second_submission upon saving, and send the email to SUCT about it. But since that is copying most of the function below I just want to call that function if possible

Reply:
Nevermind, I just copied a bit of the function. Otherwise organizers would get two emails and that's not good